molecule/alternative/playbook.yml

```diff
@@ -3,7 +3,7 @@
   hosts: all
   any_errors_fatal: true
   roles:
-    - ansible-prometheus
+    - cloudalchemy.prometheus
```

Comment:
We have pre-setup steps that create a symlink to the role name in order to handle this. But, I guess molecule now does this for us. I wonder if this is a new feature.

```
INFO     Using /home/circleci/.ansible/roles/cloudalchemy.prometheus symlink to current repository in order to enable Ansible to find the role using its expected full name.
```

Reply:
a new feature...   And, for whatever the reason, the ansible-prometheus name wasn't being found.

Reply:
Yea, I think the search path changed, in `cloudalchemy/ansilbe-node-exporter`, the symlink setup is different.